src/ui/components/Overlay/index.js

```diff
@@ -39,8 +49,12 @@ export default class Overlay extends React.Component {
       <div className={classNames('Overlay', className, {
         'Overlay--visible': this.state.visible,
       })} ref={(ref) => { this.overlayContainer = ref; }}>
-        <div className="Overlay-background"
-          ref={(ref) => { this.overlayBackground = ref; }} />
+        <div
+          onClick={this.onClickBackground}
+          onTouchEnd={this.onClickBackground}
```

Comment:
I'm actually not sure if I need this. It looks like React maybe handles this internally for `onClick` and even works around some iOS bugs. I guess I should try it and see.